index.js

```diff
@@ -96,7 +97,7 @@ module.exports = function (sails) {
                 modelDef = models[modelName];
                 sails.log.verbose('Loading model \'' + modelDef.globalId + '\'');
 
-                connectionName = modelDef.options.connection || sails.config.models.connection;
+                connectionName = modelDef.datastore || sails.config.models.datastore || 'default';
```

Comment:
in v1.0 datastore is now located in model definition root? (instead of .options, like in v0.12) ?

Reply:
I believe so http://next.sailsjs.com/documentation/concepts/models-and-orm/model-settings#?datastore